docs/3.x.x/en/plugin-development/utils.md

```diff
@@ -2,6 +2,63 @@
 
 Strapi provides helpers so you don't have to develop again and again the same generic functions.
 
+## Auth
+
+`auth.js` lets you get, set and delete data in either the browser's `localStorage` or `sessionStorage`.
+
+### Methods
+
+| Name | Description |
```

Comment:
The table is not displayed properly: https://github.com/strapi/strapi/blob/add-documentation/docs/3.x.x/en/plugin-development/utils.md